src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Common/Builder.cs

```diff
@@ -137,12 +137,20 @@ string GetVisualStudio2017Directory ()
 			}
 		}
 
+		static readonly string[] UnixNetSdkSearchPaths = {
+			Path.Combine ("/Library", "Frameworks", "Mono.framework", "Versions", "Current", "lib", "mono", "msbuild"),
+		};
+
 		public string MicrosoftNetSdkDirectory {
 			get {
 				string path;
 				if (IsUnix) {
-					path = System.IO.Path.Combine ("/usr", "local", "share", "dotnet", "sdk");
```

Comment:
Should this path *also* be present in `UnixNetSdkSearchPaths`?

Linux, for instance, won't have `/Library/...`, but *may* have `/usr/local/share/dotnet`...

Alternatively, should we use a more "robust" mechanism to find that search path?  For example, invoke `msbuild` to print out the value of the `$(FrameworkSDKRoot)` or `$(MSBuildToolsRoot)` properties?

```xml
<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
  <Target Name="Default">
    <Message Text="$(FrameworkSDKRoot)" Importance="High" />
  </Target>
</Project>
```

Then execute as `msbuild /v:minimal /nologo Path.targets` -- and `.Trim()` the output -- to get the desired *system* path:

```
$ msbuild X2.targets /v:minimal /nologo
  /Library/Frameworks/Mono.framework/Versions/5.16.0/lib/mono/msbuild/
```